tests/TestHarness/Node.py

```diff
@@ -500,7 +500,7 @@ def checkBlockForTransactions(self, transIds, blockNum):
 
     def waitForTransactionsInBlockRange(self, transIds, startBlock=2, maxFutureBlocks=None):
         lastBlockProcessed = startBlock
-        overallFinalBlock = self.getHeadBlockNum()
+        overallFinalBlock = sys.maxsize
```

Comment:
I still don't really like setting this to `sys.maxsize` as it causes a situation where once could easily create an infinite wait.  With the original `self.getHeadBlockNum()`, the damage of failing to set `maxFutureBlocks` means you basically don't wait and just evaluate all currently available blocks.

Reply:
Right but that is logically equal to 0. I'll simplify the logic and change it how you requested.